.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,179 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install gcc
+      run: |
+        sudo apt-get update
+        sudo apt-get install -yq gcc
+        sudo apt-get install -yq gfortran
+    - name: Install kim-api
+      run: |
+        export KIMPY_DIR=${PWD}
+        export KIM_API_VERSION="2.2.0"
+        cd $KIMPY_DIR && cd ..
+        wget http://s3.openkim.org/kim-api/kim-api-$KIM_API_VERSION.txz
+        tar Jxvf kim-api-$KIM_API_VERSION.txz
+        cd kim-api-$KIM_API_VERSION
+        mkdir build && cd build
+        cmake .. -DCMAKE_BUILD_TYPE=Release
+        make -j2 
+        sudo make install
+        sudo ldconfig
+        cd $KIMPY_DIR
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install ase
+        python -m pip install pytest
+    - name: Install
+      run: |
+        python -m pip install .
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        cd tests
+        pytest
+  mac:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.9]
```

Comment:
Any reason only run on 3.6 and 3.9? GH Actions number of runs limits?

Reply:
@mjwen no, just runtime reason when I was testing. Here, I was testing and for windows, using package-config had some problem that is why I forgot to add these back.